src/main/steps/switch-to-sole-application/content.ts

```diff
@@ -8,8 +8,12 @@ const en = ({ partner }) => ({
   cancel: 'Cancel',
 });
 
-// @TODO translations
-const cy: typeof en = en;
+const cy: typeof en = ({ partner }) => ({
+  title: 'Creu cais newydd',
+  newApplicationDisclaimer: `Bydd ${partner} yn colli mynediad i'ch cais ar y cyd. Byddant yn cael e-bost yn cadarnhau hyn.`,
+  create: 'Creu cais newydd',
+  cancel: 'Cancel',
```

Comment:
missing translation?

Reply:
Hi Lara, I added the missing translation.